### PR TITLE
Update Hindi Translations to match template (#824)

### PR DESCRIPTION
**Pull request by WildxHV**
Fixes # (issue)
#824 

## Type of change

Please delete options that are not relevant.

- [ ] New Question
- [x] Revision of an existing question
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate) 

# Checklist:

- [x] My content follows the style guidelines of this project
- [x] I have performed a self-review of my own content

Pull requests should be thought of as a conversation. There will be some back and forth when trying to get code merged into this or any other project. With all but the simplest changes you can and should expect that the maintainers of the project will request changes to your code. Please be aware of that and check in after you open your PR in order to get your code merged in cleanly. 

Thanks!

### CODE REVIEW

File: src/translations/hindi/README.md [hunk 1]
```diff
@@ -33,139 +29,160 @@ lang: hi
...
-* CSS डिस्प्ले प्रॉपर्टी क्या है और क्या आप इसके उपयोग के कुछ उदाहरण दे सकते हैं?
-* इनलाइन और इनलाइन-ब्लॉक के बीच क्या अंतर है?
-* `nth-of-type()` और `nth-child ()` सेलेक्टर्स के बीच क्या अंतर है?
-* `relative`, `fixed`, `absolute` और `statically positioned element` के बीच क्या अंतर है?
-* मौजूदा सीएसएस फ्रेमवर्क का आपने स्थानीय स्तर पर या उत्पादन में उपयोग किया है| आप उन्हें कैसे बदलेंगे / सुधारेंगे?
-* क्या आपने CSS ग्रिड का उपयोग किया है?
-* वेबसाइट को रेस्पॉन्सिव बनाने और मोबाइल फर्स्ट स्ट्रेटेजी के बीच क्या अंतर है?
-* क्या आपने कभी रेटिना ग्राफिक्स के साथ काम किया है? यदि हां, तो आपने कब और किन तकनीकों का उपयोग किया था?
-* क्या कोई कारण है कि आप `position: absolute` के बजाय `translate()` का उपयोग करना चाहते हैं, या इसके विपरीत? और क्यों? 
+[⬆] <a name='css'>CSS प्रश्न:</a>
```
Comment (roblarsen) [inline src/translations/hindi/README.md:84]: Remove the [⬆]

File: src/translations/hindi/README.md [hunk 2]
```diff
@@ -33,139 +29,160 @@ lang: hi
...
+CSS चयनक्रम (selector specificity) क्या है और यह कैसे काम करता है?
```
Comment (roblarsen) [inline src/translations/hindi/README.md:85]: There are no bullets in this section

File: src/translations/hindi/README.md [hunk 3]
```diff
@@ -33,139 +29,160 @@ lang: hi
...
-* कंस्ट्रक्टर के मेथड में एरो सिंटेक्स को इस्तेमाल करने का क्या फायदा है?
-* हायर-आर्डर फ़ंक्शन की परिभाषा क्या है?
-* क्या आप ऑब्जेक्ट या ऐरे को डीस्ट्रक्चर करने का उपाय बता सकते है ?
-* क्या आप ES6 टेम्प्लेट लिटरल्स के साथ एक स्ट्रिंग बनाने का एक उदाहरण दे सकते हैं?
-* क्या आप करी फंक्शन का उदाहरण दे सकते हैं और यह सिंटैक्स लाभ क्यों प्रदान करता है?
-* `spread` सिंटैक्स का उपयोग करने के क्या लाभ हैं और यह बाकी सिंटैक्स से कैसे अलग है?
-* आप फाइलस को कोड के बीच कैसे शेयर करेंगे?
-* आप स्टेटीक क्लास मेंबरस को क्यु बनाना चाहेंगे?
-* नीचे दिए गए कोड को काम करवाए :
+#### [⬆] <a name='js'>JS प्रश्न:</a>
```
Comment (roblarsen) [inline src/translations/hindi/README.md:122]: remove the [⬆]

File: src/translations/hindi/README.md [hunk 4]
```diff
@@ -33,139 +29,160 @@ lang: hi
...
+घटना अवलेपन (event delegation) को समझाएं।
```
Comment (roblarsen) [inline src/translations/hindi/README.md:123]: there are no bullets in this section

**Review by roblarsen — CHANGES_REQUESTED** [review]

**Comment by roblarsen** [comment]
Thanks so much! Just a few formatting changes and then I can review

**Comment by WildxHV** [comment]
Updated the requested changes

File: src/translations/hindi/README.md [hunk 5]
```diff
@@ -29,165 +25,179 @@ lang: hi
...
+* क्या आप ES6 टेम्पलेट लिटरल्स का उपयोग करके एक स्ट्रिंग उत्पन्न करने का एक उदाहरण दे सकते हैं?
+* क्या आप करी फ़ंक्शन का एक उदाहरण और इस सिंटैक्स के फ़ायदे के बारे में बता सकते हैं?
+* 'spread सिंटैक्स' का उपयोग करने के फ़ायदे क्या हैं और यह 'rest सिंटैक्स' से कैसे अलग है?
+* फ़ाइलों के बीच कोड साझा कैसे किया जा सकता है?
+* आपको स्टेटिक क्लास सदस्य (static class members) बनाने की क्या आवश्यकता हो सकती है?
+* 'while' और 'do-while' लूप्स के बीच क्या अंतर है?
+* एक प्रॉमिस (promise) क्या होता है? कहाँ और कैसे आप प्रॉमिस का उपयोग करेंगे?
+* JavaScript के साथ कोडिंग करते समय ऑब्जेक्ट ओरिएंटेड प्रोग्रामिंग (Object Oriented Programming) के सिद्धांतों का उपयोग करने की बात करें।
+
+## कोडिंग प्रश्न
```
Comment (roblarsen) [inline src/translations/hindi/README.md:169]: #### [[ ⬆ ]] ( #toc ) < a name = ' code-questions' >कोडिंग प्रश्न</ a >

**Review by roblarsen — APPROVED** [review]
One last formatting change and then we're good.

File: src/translations/hindi/README.md (hunk 5 above)
Comment (WildxHV) [reply src/translations/hindi/README.md]: Updated this as well

**Pull request merged**